core/src/main/java/com/ritense/valtimo/camunda/ProcessApplicationStartedEventListener.java

```diff
@@ -40,7 +44,10 @@ public ProcessApplicationStartedEventListener(ApplicationEventPublisher applicat
     public void engineStarted(ProcessApplicationStartedEvent event) {
         logger.debug("{} - handle - processApplicationStartedEvent", Thread.currentThread().getName());
         AuthorizationContext.runWithoutAuthorization(camundaProcessService::getDeployedDefinitions)
-            .forEach(processDefinition -> applicationEventPublisher.publishEvent(new ProcessDefinitionAvailableEvent(processDefinition.getId())));
+            .forEach(processDefinition -> withLoggingContext(Map.of(CamundaProcessDefinition.class.getCanonicalName(), processDefinition.getId()), true, () -> {
```

Comment:
We see this a lot 
```withLoggingContext(Map.of(CamundaProcessDefinition.class.getCanonicalName(), processDefinition.getId())``` 

Maybe we should create an extension function to take a class and output this?

Reply:
Added